addons/io/org.openhab.io.hueemulation/src/main/java/org/openhab/io/hueemulation/internal/HueEmulationService.java

```diff
@@ -286,9 +286,12 @@ public void onReadyMarkerRemoved(ReadyMarker readyMarker) {
 
     @Override
     public synchronized void onReadyMarkerAdded(ReadyMarker readyMarker) {
-        if (started || !"org.eclipse.smarthome.model.core".equals(readyMarker.getIdentifier())) {
+        if (started || !("org.openhab.core.model.core".equals(readyMarker.getIdentifier()) || 
+			"org.eclipse.smarthome.model.core".equals(readyMarker.getIdentifier()))) {
```

Comment:
So the check for "org.eclipse.smarthome.model.core" should be removed because the bundle got renamed during the ESH to OHC migration. Also the debug logging needs to be cleaned up (commented/wrong indentation).

Reply:
It perfectly makes change to check for both if the jar should be usable by OH 2.4 and OH 2.5 after milestone 1

Reply:
It's very nice to be backwards compatible. But AFAIK we don't do this with other bundles and we don't make any guarantee that this works. Users are at their own risk using these non-supported bundle combinations. It's already more complicated now with bnd all dependencies are defined  as bundles in the Karaf features and it will probably get even more complicated when https://github.com/openhab/openhab-core/pull/595 gets merged.